code/modules/mob/living/carbon/human/species.dm

```diff
@@ -55,6 +57,9 @@
 	var/siemens_coeff = 1 //base electrocution coefficient
 	var/damage_overlay_type = "human" //what kind of damage overlays (if any) appear on our species when wounded?
 	var/fixed_mut_color = "" //to use MUTCOLOR with a fixed color that's independent of dna.feature["mcolor"]
+	var/reagent_tag = PROCESS_ORGANIC //Used for metabolizing reagents. We're going to assume you're a meatbag unless you say otherwise.
+	var/species_gibs = "human"
+	var/allow_numbers_in_name // Can this species use numbers in it's name?
```

Comment:
❄️ ;)

Reply:
`comment entirely unnecessary`

Reply:
a) this is the opposite of snowflaking
b) its*